apix-impl/src/main/java/eu/xenit/apix/alfresco/people/PeopleService.java

```
@@ -127,7 +127,7 @@ private AuthorityType AuthorityTypeToAlfresco(eu.xenit.apix.people.AuthorityType
 
     @Override
     public Person GetPerson(String userName) {
-        return GetPerson(c.apix(alfrescoPersonService.getPerson(normalizeUserName(userName))));
+        return GetPerson(c.apix(alfrescoPersonService.getPersonOrNull(normalizeUserName(userName))));
```

I think this should throw an exception when a person does not exist.